master/static/srv/proto_get_trials_plus.sql

```diff
@@ -128,6 +128,13 @@ SELECT
       WHERE tt.run_id = t.id
       ORDER BY ta.start_time
   ) sub_tasks)) AS task_ids,
+  (
+    (SELECT json_agg(log_signal) FILTER (WHERE log_signal IS NOT NULL) FROM (
+      SELECT ta.log_signal FROM run_id_task_id tt
+      JOIN tasks ta ON tt.task_id = ta.task_id
+      WHERE tt.run_id = t.id
+      ORDER BY ta.start_time
+  ) log_signals)) AS log_signals,
```

Comment:
As far as I understand, one trail has only one task, so there will be only one log_signal.